zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -71,3 +76,25 @@
     .skip_configuration()
     .add_to_registry()
 )
+
+(
+    TuyaQuirkBuilder("_TZE200_pay2byax", "TS0601")  # Cusam ZG-102ZL
+    .applies_to("_TZE200_n8dljorx", "TS0601")
```

Comment:
I'd lean towards leaving `tuya_battery` in, from the incident, the device does report the value on DP 2 and that's what z2m is doing also.